portality/tasks/journal_in_out_doaj.py

```diff
@@ -53,10 +54,18 @@ def run(self):
 
         for journal_id in journal_ids:
             job.add_audit_message("Setting in_doaj to {x} for journal {y}".format(x=str(in_doaj), y=journal_id))
-
+            # ~~->Journal:Model~~
             j = models.Journal.pull(journal_id)
+            # ~~->Account:Model~~
+            account = models.Account.pull(job.user)
             if j is None:
                 raise RuntimeError("Journal with id {} does not exist".format(journal_id))
+            if not in_doaj:
+                job.add_audit_message("Rejecting all associated update requests")
+                # ~~->Application:Service~~
+                svc = DOAJ.applicationService()
+                ur = svc.reject_update_request_of_journal(j.id, account)
+                job.add_audit_message("Update request {x} automatically rejected".format(x=ur))
```

Comment:
`ur` can be none, that needs to be caught, and the audit messages need to make sense.  Either don't add the audit messages at all if there's no UR, or make the second message say that (I'd lean towards the former)